korebuild.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/release/2.1/tools/korebuild.schema.json",
-  "channel": "release/2.1",
+  "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
```

Comment:
You have changes from dev in your PR to release/2.1

Reply:
I changed the base without @mikaelm12 rebasing.

Reply:
Oh yeah. Didn't make this change on release/2.1 
I'll fix this